resources/views/admin/speaker/view.twig

```diff
@@ -15,6 +15,15 @@
             <div>
                 {% if speaker.company %}<span class="mr-3"><i class="fa fa-building"></i> {{ speaker.company }}</span>{% endif %}
                 {% if speaker.twitter %}<span class="mr-3"><a class="hover:text-brand" target="_blank" href="{{ speaker.twitterUrl }}" rel="noopener noreferrer"><i class="fa fa-twitter"></i> @{{ speaker.twitter }}</a></span>{% endif %}
+                {% if speaker.joindInUsername %}
+                    <span class="mr-3">
+                        <a class="hover:text-brand" target="_blank"
+                           href="{{ speaker.joindInUrl }}"
```

Comment:
localheinz 7 hours ago

> Unless I am terribly mistaken, and to answer your question from the body of the pull request, I think it should be possible to use
> 
> ```diff
> diff --git a/resources/views/admin/speaker/view.twig b/resources/views/admin/speaker/view.twig
> index ace150ce..3d9e23fe 100644
> --- a/resources/views/admin/speaker/view.twig
> +++ b/resources/views/admin/speaker/view.twig
> @@ -18,9 +18,9 @@
>                  {% if speaker.getJoindInUsername() %}
>                      <span class="mr-3">
>                          <a class="hover:text-brand" target="_blank"
> -                           href="https://joind.in/user/{{ speaker.getJoindInUsername() }}"
> +                           href="https://joind.in/user/{{ speaker.joindInUsername }}"
>                             rel="noopener noreferrer">
> -                            <i class="fa fa-commenting-o"></i> {{ speaker.getJoindInUsername() }}
> +                            <i class="fa fa-commenting-o"></i> {{ speaker.joindInUsername }}
>                          </a>
>                      </span>
>                  {% endif %}
> ```
> 
> here instead. Have you tried?
> 
> Maybe it also makes sense to extract a method and shift the construction of the URL to `SpeakerProfile`, what do you think?

chartjes 6 hours ago

> I like the idea of having `SpeakerProfile` generate this too since I recently merged in a PR that moved the generation of Twitter URL's to `SpeakerProfile` as well

chrisforrence 6 hours ago

> Haha, I spent half an hour on stream trying to figure out why speaker.joindInUsername hadn't been working :p I can certainly give it another crack tonight though!

> I can definitely be on board with either constructing the URL through SpeakerProfile or on a JoindIn helper (and something similar for Twitter as well)

chrisforrence 6 hours ago

> Ah, well then, that answers the second part!

Reply:
I've got the SpeakerProfile now creating a joind.in URL as a getter, and I figured out why I wasn't able to get it working before (I had been trying to refer to it with `joindin_username` with the underscore, not `joindInUsername`). Should be good now!